include/Note.h

```diff
@@ -209,8 +209,7 @@ class EXPORT Note : public SerializingObject
 
 
 protected:
-	virtual void saveSettings( QDomDocument & _doc,
-							QDomElement & _parent );
+	virtual void saveSettings( QDomDocument & doc, QDomElement & parent );
 	virtual void loadSettings( const QDomElement & _this );
```

Comment:
`_this` seems badly named. I realise you didn't introduce it, but why not fix it too while you're on it. :-)